README.md

```diff
@@ -6,3 +6,56 @@ This module is used to create an ECS service running phpmyadmin.
  - Create target group for ALB to route based on hostname
  - Create task definition and ECS service for phpmyadmin
  - Create Cloudflare DNS record
+
+## Provider dependencies
+
+### aws
+
+[providers/hashicorp/aws](https://registry.terraform.io/providers/hashicorp/aws)
+
+example configuration:
+
+```hcl
+provider "aws" {
+  region     = var.aws_region
+  access_key = var.aws_access_key_id
+  secret_key = var.aws_secret_access_key
+}
+```
+
+### cloudflare
+
+[providers/cloudflare/cloudflare](https://registry.terraform.io/providers/cloudflare/cloudflare)
+
+example configuration:
+
+```hcl
```

Comment:
Can we show two examples, one that uses just api_token and another that uses email, key, account id? I don't recall if account id is needed in the token use case or not.